msgpack_test.go

```diff
@@ -1,4 +1,4 @@
-package msgpack_test
+package msgpack
```

Comment:
The build tag allow to build outside of official tree, but the tests still run from official tree because of `import "gopkg.in/vmihailenco/msgpack.v2"`.